[dist_optim] Add distributed functional Adamax optimizer

wanchaol opened the pull request:
Summary: Add distributed functional Adamax optimizer, to support in TorchScript

Differential Revision: D26696540

pritamdamania87 reviewed (commented):
Changes LGTM, will let someone on the torch.optim side stamp this though.

Commit:
[optim] refactor rprop to use functional API

Summary: as titled

Differential Revision: D27703877

fbshipit-source-id: 485f112c2f38a583d6296e72ed0676f76722f992

Commit:
[optim] refactor adamax to use functional API (#52639)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/52639

Differential Revision: D26561017

fbshipit-source-id: 88200e282defe5e7e08ef84fbea2a128c75fcae8

Commit:
[dist_optim] Add distributed functional Adamax optimizer (#55598)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/55598

Add distributed functional Adamax optimizer, to support in TorchScript

Differential Revision: D26696540

fbshipit-source-id: c81c68acc5e6c108209602ede09cd9d7596fd5fc

wanchaol commented:
messed up with export, close this and re-export the diff
